src/pages/authors/[slug].js

```diff
@@ -41,10 +43,10 @@ export async function getStaticPaths() {
   const { authors } = await getAllAuthors();
 
   const paths = authors.map((author) => {
-    const { slug } = author;
+    const { name } = author;
     return {
       params: {
-        slug,
+        slug: parameterize(name),
```

Comment:
maybe it would be nice to create yet an additional function in `lib/users` called something like `userSlugByName`, that way we can keep any use of `parameterize` all in one spot and if we ever need to update it to something else, we can make all the change sin that one file. thoughts?

Reply:
Agree!, parameterize at one spot sounds way better